src/Docker/Manager/ImageManager.php

```diff
@@ -104,15 +104,22 @@ public function find($repository, $tag = 'latest')
      * @throws \Docker\Exception\UnexpectedStatusCodeException
      * @throws \GuzzleHttp\Exception\RequestException
      *
-     * @return ImageManager
+     * @return @return json data from docker inspect
      */
     public function inspect(Image $image)
     {
         try {
-            $response = $this->client->get(['/images/{id}/json', ['id' => $image->__toString()]]);
+            # Images need not have a name and tag,(__toString() may return ':')
+            # so prefer an id hash as the key
+            if (null != $image->getId()) {
+              $id = $image->getId();
+            } else {
+              $id = $image->__toString();
+            }
+            $response = $this->client->get(['/images/{id}/json', ['id' => $id]]);
```

Comment:
Before line 119.

More generally, after any structure block (`if`, `for`, `while`, etc) and before any `return` statement. I'll add a documentation for the coding style soon :)